Commands/Create.js

```diff
@@ -15,36 +15,13 @@ class Create extends Command {
   }
 
   async execute(Interaction) {
-    // This is a development test command 
```

Comment:
Do you think this might take time to respond (>3 seconds?). If so you might want to add:
```Javascript
await Interaction.defer();
```
and then change `.reply()` to `.editReply()`

Reply:
Good to keep in mind, in theory shouldn't be needed but appreciate the future/potential thought 🎉